### ISSUE TITLE
Something is up with TTVs

### ISSUE THREAD

**Issue by KoboldCommando** [Monkestation/MonkeStation#135]
There's not much information and I wasn't able to find anything useful codediving.  But someone has been attempting to learn maxcaps and has been VERY frustrated because things just aren't working right.  The booms are happening but the magnitude isn't what it should be.

**Comment by KoboldCommando** [comment]
self defense TTV — Today at 1:33 AM
A bee admin told me to say "update auxmos.dll"

**Comment by RareThrumbo** [comment]
To provide some context, the following bomb was fired on both Monke and Bee. I provided screenshots in the Bee Discord, somebody opened a private instance of live Bee code and replicated my bomb and had very different results than I did.

Oxygen Tank:
Moles: 494.48 mol
Volume: 70 L
Pressure: 2533 kPa
Oxygen: 66.64% (329.53 mol)
Tritium: 33.36% (164.96 mol)
Temperature: 43.15 K

Plasma Tank:
Moles: 0.34 mol
Volume: 70 L
Pressure: 2533 kPa
Plasma: 100% (0.34 mol)
Temperature: 61871.5C (62144.6K)

On Beestation, this produced a bomb with 50,000 research points.
On Monkestation, this produced a bomb with ~39,000 research points.

So there is some discrepancy between Monke atmos/TTV and Bees.
